### PR TITLE
Add cmd_vel_topic param

### PR DESCRIPTION
**Pull request by KmakD**
plus some small fixes in naming and accel values

### CODE REVIEW

File: webui-ros-joystick/launch/panther_webui.launch [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
   <arg name="node_name" default="node_server" />
   <arg name="wait_nodes" default="" />
+  <arg name="cmd_vel_topic" default="/cmd_vel" />
   <arg name="max_lin_vel" default="2.0" />
   <arg name="max_ang_vel" default="1.0" />
-  <arg name="max_lin_accel" default="2.0" />
-  <arg name="max_ang_accel" default="2.0" />
+  <arg name="max_lin_accel" default="20.0" />
```
Comment (TopRamen1) [inline webui-ros-joystick/launch/panther_webui.launch:8]: Was this intentional?

File: webui-ros-joystick/nodejs/main.js [hunk 2]
```diff
@@ -137,31 +137,32 @@ rosnodejs
...
+      geometryMsgs.Twist,
+      defaultPublisherOptions
+    );
 
     if (eStopPresent) {
       eStop = true;
 
       eStopSubscriber = rosNode.subscribe(
-        "/e_stop",
+        "e_stop",
```
Comment (TopRamen1) [inline webui-ros-joystick/nodejs/main.js:159]: The e_stop changes are not related to the task, are they intentional?

File: webui-ros-joystick/launch/webui.launch [hunk 3]
```diff
@@ -2,19 +2,21 @@
...
+    <arg name="cmd_vel_topic" default="/cmd_vel" />
     <arg name="max_lin_vel" default="1.0" />
     <arg name="max_ang_vel" default="1.0" />
     <arg name="max_lin_accel" default="2.0" />
     <arg name="max_ang_accel" default="2.0" />
     <arg name="e_stop" default="false" />
 
     <node pkg="webui-ros-joystick" type="node_server.sh" name="$(arg node_name)" output="screen">
+        <param name="wait_nodes" value="$(arg wait_nodes)" />
+        <param name="cmd_vel_topic" value="$(arg cmd_vel_topic)" />
```
Comment (Kotochleb) [inline webui-ros-joystick/launch/webui.launch:14]: There is remapping in ROS for a reason. Passing topic name as a param is not the best practice

File: webui-ros-joystick/launch/webui.launch [hunk 4]
```diff
@@ -2,19 +2,21 @@
 
     <arg name="node_name" default="node_server" />
     <arg name="wait_nodes" default="" />
```
Comment (Kotochleb) [inline webui-ros-joystick/launch/webui.launch:4]:
```suggestion
    <arg name="await_nodes" default="" />
```

File: webui-ros-joystick/launch/panther_webui.launch (hunk 1 above)
Comment (KmakD) [reply webui-ros-joystick/launch/panther_webui.launch:8]: Yes, that should be default configuration but was missed in previous pr

**Review by Kotochleb — CHANGES_REQUESTED** [review]

File: webui-ros-joystick/nodejs/main.js (hunk 2 above)
Comment (KmakD) [reply webui-ros-joystick/nodejs/main.js:159]: I know, but as I was looking at topics naming i changed that so namespaces could be used. So I added this quick fix :P

File: webui-ros-joystick/launch/webui.launch (hunk 3 above)
Comment (KmakD) [reply webui-ros-joystick/launch/webui.launch:14]: Generally yes, but using param for this topic is more versatile e.g. useful when using yaml config files. Adding params for crucial topics is used for example in Nav2.

File: webui-ros-joystick/launch/webui.launch (hunk 4 above)
Comment (KmakD) [reply webui-ros-joystick/launch/webui.launch:4]: Old naming, would require changes in few files

File: webui-ros-joystick/launch/panther_webui.launch (hunk 1 above)
Comment (TopRamen1) [reply webui-ros-joystick/launch/panther_webui.launch:8]: Generally we should do things like this in separate PRs

File: webui-ros-joystick/nodejs/main.js (hunk 2 above)
Comment (TopRamen1) [reply webui-ros-joystick/nodejs/main.js:159]: Same as above

**Comment by TopRamen1** [comment]
Will be merging as this is an urgent change. As we discussed with @Hubert424 if @Kotochleb does not agree with them we need to have this discussion when he is back
